src/components/VForm/VForm.js

```diff
@@ -78,11 +75,13 @@ export default {
       })
     },
     validate () {
-      return !this.inputs.filter(input => input.validate(true)).length
+      return !this.inputs.filter(input => input.validate(true))
```

Comment:
What is the reasoning for this change?

Reply:
Oops, thanks for pointing that out. That wasn't meant to be pushed.